kolibri/core/discovery/test/test_connections.py

```diff
@@ -141,6 +141,12 @@ def test_okay__dynamic(self):
         update_network_location(self.mock_location)
         self.assertNotEqual(self.mock_location.last_known_ip, "192.168.101.101")
 
+    def test_okay__static(self):
+        self.mock_location.dynamic = False
+        update_network_location(self.mock_location)
+        self.assertEqual(self.mock_location.last_known_ip, "192.168.101.101")
+        self.assertIn(self.mock_location.is_local, [True, False])
```

Comment:
Seems like "192.168.101.101" should always be a local IP address?

Reply:
It doesn't seem to be when run on `python 2.7`. Maybe  underlying issue with some parts of the porting? Let me try to fiddle with this and revert.

Reply:
I saw [this issue](https://github.com/learningequality/kolibri/issues/10363) you added. The main reason to vendor the library is to support 2.7, which I'm sure required you to make compatibility tweaks to the vendored code. But if the functionality that we need isn't working in 2.7, then that seems problematic!

Reply:
Issue has been resolved and test made more specific (`self.assertTrue(self.mock_location.is_local)`)